tests/integration/framework/process/exec/iowriter/iowriter_test.go

```diff
@@ -86,19 +91,56 @@ func TestWrite(t *testing.T) {
 
 func TestClose(t *testing.T) {
 	t.Run("should flush and close", func(t *testing.T) {
-		logger := &mockLogger{t: t}
+		logger := &mockLogger{t: t, failed: true}
+		writer := New(logger, "proc").(*stdwriter)
+		writer.Write([]byte("test"))
+		writer.Close()
+
+		assert.Equal(t, 0, writer.buf.Len())
+		_ = assert.Equal(t, 1, len(logger.msgs)) &&
```

Comment:
Can not use two single assert lines? Why need `&&`

Reply:
I presume it's to avoid a panic if len == 0. If so, trying to access `logger.msgs[0]` would panic

Reply:
Yep exactly, it's to prevent the panic but don't want to use `require` necessarily because we want to continue the test. ^